plugins/mediatek-scaler/fu-mediatek-scaler-device.c

```diff
@@ -402,31 +436,46 @@ fu_mediatek_scaler_device_probe(FuDevice *device, GError **error)
 						      error))
 		return FALSE;
 
-	/* prioritize DDC/CI commands in display controller */
-	if (!fu_mediatek_scaler_device_set_ddc_priority(device, FU_DDCCI_PRIORITY_UP, error))
+	/* get vid and pid from PCI bus */
+	udev_parent = fu_udev_device_get_parent_with_subsystem(FU_UDEV_DEVICE(device), "pci");
+	if (udev_parent == NULL)
 		return FALSE;
 
-	/* mediatek display is connected */
-	if (!fu_mediatek_scaler_display_is_connected(device, error))
+	if (!fu_device_probe(FU_DEVICE(udev_parent), error))
 		return FALSE;
 
-	/* set hardware version */
-	hw_ver = fu_mediatek_scaler_device_get_hardware_version(device, error);
-	if (hw_ver == NULL)
+	vid_u16 = fu_udev_device_get_subsystem_vendor(udev_parent);
+	pid_u16 = fu_udev_device_get_subsystem_model(udev_parent);
+
+	/* supported host system only */
+	if (!(vid_u16 != 0x1028) && ((pid_u16 != 0x0c88) || (pid_u16 == 0x0c8a)))
```

Comment:
You need to set error here.

Reply:
> You need to set error here.

Thanks for reviewing here, the vcp test has been mreged to another brach. I will close this PR.